src/librbd/image_watcher/NotifyLockOwner.cc

```diff
@@ -80,6 +80,7 @@ void NotifyLockOwner::handle_notify(int r) {
     decode(response_message, iter);
 
     r = response_message.result;
+    ldout(cct, 1) << " client responded with r=" << r << dendl;
```

Comment:
nit: I think the log level 1 is too low for a message like this. I would use 10 or 20.

Reply:
makes sense +1